integration-tests/e2e/kubetest/results_evaluator.go

```diff
@@ -66,70 +66,111 @@ func writeSummaryToFile(summary Summary) {
 }
 
 func analyzeJunitXMLsEnrichSummary(junitXMLFilePaths []string, summary *Summary) error {
-	var mergedJunitXmlResult = &JunitXMLResult{FailedTests: 0, ExecutedTests: 0, DurationFloat: 0, SuccessfulTests: 0, DurationInt: summary.TestsuiteDuration}
-	testcaseNameToTestcase := make(map[string]TestcaseResult)
-	failureOccurrences := make(map[string]int)
+	var mergedJunitXmlResult = &JunitXMLResult{DurationInt: summary.TestsuiteDuration}
```

Comment:
```suggestion
	mergedJunitXmlResult := &JunitXMLResult{DurationInt: summary.TestsuiteDuration}
```
nit/ just to be in sync with the assignments in 71-74